programs/steward/src/instructions/idle.rs

```diff
@@ -14,6 +16,10 @@ pub struct Idle<'info> {
         bump
     )]
     pub state_account: AccountLoader<'info, StewardStateAccount>,
+
+    /// CHECK: Account owner checked, account type checked in get_validator_stake_info_at_index
```

Comment:
nit: account type checked in `get_validator_list_length`, and owner is not checked but address is verified by being set in config

Reply:
Done